.github/workflows/ci-test-frontend.yml

```
@@ -0,0 +1,25 @@
+name: test_frontend
+
+on:
+  push:
+    branches:
+    tags:
+    paths:
+    - 'frontend/**'
+  pull_request:
+    paths:
+    - 'frontend/**'
+
+jobs:
+  frontend:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-node@v1
+    - run: npm ci
+      working-directory: ./frontend
+    - run: npx run-p lint test build
```

It also needs ts-check

```suggestion
    - run: npx run-p lint test build
```